src/main/java/net/foxes4life/foxclient/Main.java

```diff
@@ -41,7 +41,7 @@ void initConfig () {
         KonfigCategory misc = new KonfigCategory("misc");
         misc.addEntry("discord-rpc", true);
         misc.addEntry("discord-rpc-show-ip", true);
-        misc.addEntry("smoothzoom", true);
+        misc.addEntry("zoom", true);
```

Comment:
Why is this changed? Having the config key as just "zoom" might be misleading since its for smooth zoom and not zoom in general

Reply:
:+1: